boa_cli/src/main.rs

```diff
@@ -117,6 +117,7 @@ impl Opt {
     }
 }
 
+/*
 arg_enum! {
```

Comment:
If this is no longer needed, can it be removed?

Reply:
Yeah i think it should be removed.